src/components/introduction.stories.mdx

```diff
@@ -68,13 +81,13 @@ const MyApp = () => {
     const response = await fetch('/my-server');
     const data = await response.json();
 
-    const joinData = {
-      meetingInfo: data.Meeting,
-      attendeeInfo: data.Attendee,
-    };
+    // Initalize the meeting session configuration
+    let meetingSessionConfiguration = new MeetingSessionConfiguration(data.JoinInfo.Meeting, data.JoinInfo.Attendee);
 
-    // Create a `MeetingSession` using `join()` function with the above data
-    await meetingManager.join(joinData);
+    // Create a `MeetingSession` using `join()` function with the meeting session configuration
+    await meetingManager.join({
```

Comment:
I will add `MeetingJoinData` here. I will not update every occurrence of this since the other syntax is also correct and this syntax is just more lines of code. It is ok to add this in intro since it will be the first thing a builder will read.